lms/djangoapps/course_api/blocks/tests/test_serializers.py

```diff
@@ -40,6 +40,8 @@ def setUp(self):
             'request': MagicMock(),
             'block_structure': self.block_structure,
             'requested_fields': ['type'],
+            'lti_url': False,
+            'block_types': ['html']
```

Comment:
I'd wait until the block_type discussion above gets resolved, but I don't see a test for when option block_types is not included.